.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Tracking issue
+about: Use this template for tracking complaints.
+title: "[DATE]: [ISSUE NAME]"
+labels: tracking issue, needs triage
+assignees: wwelling
+---
+
+1. What is the type of this issue?
+    - [ ] Client
+
+        - [ ] Documentation: documentation is missing, incomplete, incorrect, ambiguous, or incomprehensible.
+        - [ ] Functional: no longer able to satisfy use case due to newly incurred limitations.
```

Comment:
> Functional: no longer able to satisfy use case due to newly incurred limitations.
The wording is rather confusing to me.

Maybe something more along the lines of "...the expected or intended functionality is not provided...".